docs/spec/rum_v3_context.json

```diff
@@ -153,6 +153,81 @@
                         "string",
                         "null"
                     ]
+                },
+                "ni": {
+                    "description": "networkInfo",
+                    "type": [
+                        "object",
+                        "null"
+                    ],
+                    "properties": {
+                        "et": {
+                            "description": "effectiveType",
+                            "type": [
+                                "string",
+                                "null"
+                            ],
+                            "maxLength": 1024
+                        },
+                        "rtt": {
+                            "description": "roundTripTime",
+                            "type": [
+                                "integer",
+                                "null"
+                            ]
+                        },
+                        "dlk": {
```

Comment:
any reason for three letter combination instead of just `d` ? 
```
downlink -> d
downlinkMax -> dm
saveData -> sd
```

that would align with the camelCasing of the spec.

Reply:
d and sd are all already used for duration and started fields, so we would know which one refers too. we could potentially resolve ambiguity by looking at the parent field, but that would entail more complexity than what i am willing to introduce...

Reply:
Fair enough, not a big of a deal. I forgot that we dont take depth of parent in to consideration. Let keep it as it is for now.